Tools/ardupilotwaf/ap_library.py

```diff
@@ -118,6 +118,11 @@ def ap_library(bld, library, vehicle):
         for s in bld.env.AP_LIB_EXTRA_SOURCES[library]:
             src.append(bld.bldnode.find_or_declare(os.path.join('libraries', library, s)))
 
+    # include the dependencies for XRCE-DDS
+    if len(bld.env.XRCE_DEPENDENCIES):
```

Comment:
I don't see where XRCE_DEPENDENCIES is being set? I think this may be a leftover from an earlier version of the PR

Reply:
I don't see either.Removed in `1c28f067a43cd1c9de4fcb81664c3514e6b4dbae`. Build is fine without it.